infer/src/integration/SarifReport.ml

```diff
@@ -69,9 +69,8 @@ let pp_results_header fmt =
 
 let loc_trace_to_sarifbug_record trace_list =
   let file_loc filename =
-    let absolute_source_name = Config.project_root ^/ filename in
-    let file_path = "file:" ^ filename in
-    {Sarifbug_j.uri= file_path; Sarifbug_j.uriBaseId= absolute_source_name}
+    let absolute_project_root = "file:" ^ Config.project_root in
+    {Sarifbug_j.uri= filename; Sarifbug_j.uriBaseId= absolute_project_root}
```

Comment:
Alternatively, `uriBaseId` can be set to the constant `"%srcroot%"` as described [here](https://docs.oasis-open.org/sarif/sarif/v2.1.0/errata01/os/sarif-v2.1.0-errata01-os-complete.html#_Toc141790681). This helps when scanning repositories where we typically want the SARIF report to be the same (for the same commit) regardless of where the local checkout appears on the filesystem.